zkml/src/layers/transformer/softmax.rs

```diff
@@ -31,7 +31,7 @@ impl Evaluate<f32> for Softmax<f32> {
             .map(|vec| {
                 let sum = vec.iter().map(|x| x.exp()).sum::<f32>();
                 vec.iter()
-                    .map(|x| self.scale * x.exp() / sum)
+                    .map(|x| (self.scale * x).exp() / sum)
```

Comment:
I think you should include it 'self.scale` also when computing the sum in the previous line?